cmd/jaeger/docs/migration/all-in-one-metrics.md

```diff
@@ -0,0 +1,92 @@
+### Common Metrics
+
+| Metric Name | Inner Parameters |
+|-------------|------------------|
+| jaeger_query_latency | operation, result |
+| jaeger_query_responses | operation |
+| jaeger_query_requests_total | operation, result |
+### V1 Only Metrics
+
+| Metric Name | Inner Parameters |
+|-------------|------------------|
+| go_gc_duration_seconds |  |
```

Comment:
@Wise-Wizard we should investigate how these standard Go metrics are being generated in v1 and decide how we can enable them in v2.

Reply:
this is what we might want: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/runtime/runtime.go

ideally Collector would expose those automatically, but it's not implemented currently -- https://github.com/open-telemetry/opentelemetry-collector/issues/2155